AutoDuty/Managers/ActionsManager.cs

```diff
@@ -440,7 +441,20 @@ public void Revival(string _)
         {
             _taskManager.Enqueue(() => AutoDuty.Plugin.Action = "");
         }
-
+        
+        public void CameraFacing(string coords)
+        {
+            if (coords != null)
+            {
+                string[] v = coords.Split(", ");
+                if (v.Length == 3)
+                {
+                    Vector3 facingPos = new Vector3(float.Parse(v[0]), float.Parse(v[1]), float.Parse(v[2]));
```

Comment:
This needs to add invariant culture or it will error out in places that swap the . And ,

Reply:
Refer to FisnsWaypointFunctuin in line 661 of Autoduty.cs, specifically the float.parse calls and their 2nd parameter